docs/site/Model.md

```diff
@@ -407,12 +389,42 @@ class MyUserModel extends Entity {
 }
 ```
 
-### Property Decorator
+### Scope
```

Comment:
```suggestion
### Scopes
```

Reply:
The field name is in plural :p https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#scopes

Reply:
Yes, but in LB4 it only works with settings `scope`:
```ts
@model({
  settings: {
    scope: {
      'limit': 2,
    },
    idInjection: true
  }
})
```
:( that's why I made them `scope` instead of `scopes`

Reply:
I see! remembered, LB3 supports both `scopes` and `scope`:
- scopes: a collection of predefined scopes
- scope: the default scope

So your change makes perfect sense 💯